added killall process function and supporting Resources bundle folder

iMokhles opened the pull request:
add User-Defined iOSOpenDevKillProcess = process name
you can remove iOSOpenDevRespringOnInstall ;) and add iOSOpenDevKillProcess = Springboard or SpringBoard

iMokhles committed:
added killall process function

add User-Defined iOSOpenDevKillProcess = process name
you can remove iOSOpenDevRespringOnInstall ;) and add iOSOpenDevKillProcess = Springboard or SpringBoard

spencerjames commented:
You've created a dependency of `dpkg-deb` which is not available on OS X by default.

iMokhles committed:
removed dpkg-deb modifications

ursadddd reviewed (approved):
review

iMokhles committed:
create bundle from Resources folder

create Resources folder inside target's directory and place all your resources images and files there and build your project :) 
then all of those resources will be inside /Library/Application Support/TARGET_NAME/TARGET_NAME.bundle folder
